webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/services/FileManagerJaxrsServiceV2.java

```diff
@@ -137,6 +137,20 @@ public ProductTypeListResource getProductTypes() throws WebApplicationException
     }
   }
 
+  public int getTotalNumOfProducts() throws WebApplicationException {
```

Comment:
Minor: Shall we add a JavaDoc comment for the method?

Reply:
Sure. added it.